pkgs/development/python-modules/sphinxcontrib-svg2pdfconverter/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  python3Packages,
+  fetchPypi,
+}:
+
+python3Packages.buildPythonPackage rec {
+  pname = "sphinxcontrib-svg2pdfconverter";
+  version = "1.2.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit version;
+    pname = "sphinxcontrib_svg2pdfconverter";
+    hash = "sha256-+11Re2NMVilSIATFntzk2QUNiYIkCMq0UsfVL+WumCQ=";
+  };
+
+  build-system = with python3Packages; [
+    setuptools
+  ];
+
+  dependencies = [ python3Packages.sphinx ];
```

Comment:
`cairosvg` is an optional dependency. Why is it not included?

Also the package may use inkscape / rsvg-convert as external binaries. It would be nice to provide these too.